src/app/visual/sky-tree-view/angular-tree-visual.component.html

```diff
@@ -0,0 +1,221 @@
+            </sky-dropdown>
+          </sky-tree-view-context-menu>
+        </sky-tree-view-node-wrapper>
+      </ng-template>
+
+    </tree-root>
+
+  </sky-tree-view>
+</div>
+
+<h3>Cascading checkboxes</h3>
+<div
+  id="screenshot-tree-view-cascading-checkboxes"
+  class="sky-padding-even-default"
+>
+  <sky-tree-view
+    [showToolbar]="true"
+  >
+    <tree-root
+      [nodes]="nodes"
+      [options]="optionsCascading"
+      #treeCascading
+    >
+      <ng-template
+        let-index="index"
+        let-node
+        #treeNodeWrapperTemplate
+      >
+        <sky-tree-view-node-wrapper
+          [index]="index"
+          [node]="node"
+        >
+        </sky-tree-view-node-wrapper>
+      </ng-template>
+    </tree-root>
+  </sky-tree-view>
+</div>
+
+<h3>Non-cascading checkboxes</h3>
+<div
+  id="screenshot-tree-view-non-cascading-checkboxes"
+  class="sky-padding-even-default"
+>
+  <sky-tree-view
+    [showToolbar]="true"
+  >
+    <tree-root
+      [nodes]="nodes"
+      [options]="optionsNoncascading"
+      #treeNonCascading
+    >
+      <ng-template
+        let-index="index"
+        let-node
+        #treeNodeWrapperTemplate
+      >
+        <sky-tree-view-node-wrapper
+          [index]="index"
+          [node]="node"
+        >
+        </sky-tree-view-node-wrapper>
+      </ng-template>
+    </tree-root>
+  </sky-tree-view>
+</div>
+
+<h3>Checkboxes with leaf node selection only</h3>
+<div
+  id="screenshot-tree-view-leaf-selection-only"
+  class="sky-padding-even-default"
+>
+  <sky-tree-view
+    [showToolbar]="true"
+    [options]="optionsLeafNodeOnly"
+  >
+    <tree-root
+      [nodes]="nodes"
+      [options]="optionsNoncascading"
+      #treeNonCascading
+    >
+      <ng-template
+        let-index="index"
+        let-node
+        #treeNodeWrapperTemplate
+      >
+        <sky-tree-view-node-wrapper
+          [index]="index"
+          [node]="node"
+        >
+        </sky-tree-view-node-wrapper>
+      </ng-template>
+    </tree-root>
+  </sky-tree-view>
+</div>
+
+<h3>Binding to events</h3>
+<div
+  class="sky-padding-even-default"
+>
+  <sky-tree-view
+    [showToolbar]="true"
+  >
+    <tree-root
+      [nodes]="nodes"
+      [options]="optionsCascading"
+      (deselect)="onSelectionChange($event)"
+      (select)="onSelectionChange($event)"
+      #treeEvents
+    >
+      <ng-template
+        let-index="index"
+        let-node
+        #treeNodeWrapperTemplate
+      >
+        <sky-tree-view-node-wrapper
+          [index]="index"
+          [node]="node"
+        >
+        </sky-tree-view-node-wrapper>
+      </ng-template>
+    </tree-root>
+  </sky-tree-view>
+  <p>
+    <b>Selected nodes:</b>
+    {{ selectedNodeIds }}
+  </p>
+</div>
+
+<h3>Split view</h3>
+<p>Split view doesn't exist yet, but this is to demonstrate "active" styles when selecting nodes</p>
+<div
+  id="screenshot-tree-view-split-view"
+  class="sky-padding-even-default sky-split-view"
+>
+  <sky-tree-view>
```

Comment:
I may have misunderstood our conversation a few days ago, but I remember us landing on `<sky-tree-root-wrapper>` for this component's name, so there's no confusion about its purpose?